agenda/data/src/main/java/com/nibalk/tasky/agenda/data/remote/mapper/EventRemoteMapper.kt

```diff
@@ -0,0 +1,46 @@
+package com.nibalk.tasky.agenda.data.remote.mapper
+
+import com.nibalk.tasky.agenda.data.remote.dto.EventDto
+import com.nibalk.tasky.agenda.domain.model.AgendaItem
+import com.nibalk.tasky.agenda.domain.model.EventPhoto
+import com.nibalk.tasky.core.data.utils.toLocalDateTime
+import com.nibalk.tasky.core.data.utils.toLongDate
+import java.util.UUID
+
+fun AgendaItem.Event.toEventDto(): EventDto {
+    return EventDto(
+        id = id ?: UUID.randomUUID().toString(),
+        title = title,
+        description = description,
+        startAt = startAt.toLongDate(),
+        remindAt = remindAt.toLongDate(),
+        endAt = endAt.toLongDate(),
+        isHost = isHost,
+        hostId = hostId,
+        attendees = attendees.map { attendee ->
+            attendee.toEventAttendeeDto()
+        },
+        photos = photos.filterIsInstance<EventPhoto.Remote>().map { remotePhoto ->
+            remotePhoto.toEventPhotoDto()
+        },
+    )
+}
+
+fun EventDto.toAgendaItemEvent(): AgendaItem.Event {
+    return AgendaItem.Event(
+        id = id,
+        title = title,
+        description = description,
+        startAt = startAt.toLocalDateTime(),
+        remindAt = remindAt.toLocalDateTime(),
```

Comment:
I'm missing the information here how you interpret the Long timestamp. Since it's in UTC and you want to display the times in the user's local timezone, do you properly handle the conversion? If so, I'd communicate that with the function name, e.g. `startAt.asUtcToLocalDateTime()`

Reply:
Slack chat discussion, copying here for reference

> Converting from a timestamp looks good, since they’re in UTC by default. But when converting to a timestamp you use the local user’s timezone. Imagine you save such a timestamp in a non-UTC timezone and the user travels across timezones. When writing the timestamp to the DB you used timezone A, but if they’re in a different zone then, you’ll interpret it as timezone B which will lead to incorrect times.
ZonedDateTime should be used if you explicitly want to include the timezone information in the date time instance. LocalDateTime is always meant to represent a specific local time without any timezone information. I would work with ZonedDateTime references for apps where timezone is important.